include/boost/date_time/date_parsing.hpp

```diff
@@ -9,13 +9,16 @@
  * $Date$
  */
 
+#include <map>
```

Comment:
right nevermind on the other question -- circling back I see why you moved the includes not.  Of course the correct order for truly self contained headers is the std lib headers need to go after the boost headers.  But this is fine.